src/main/java/it/gov/pagopa/payhub/ionotification/event/producer/NotificationProducer.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.payhub.ionotification.event.producer;
+
+import it.gov.pagopa.payhub.model.generated.NotificationQueueDTO;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.stereotype.Component;
+
+@Component
+public class NotificationProducer {
+
+    @Value("${spring.cloud.stream.bindings.notificationQueue-out-0.binder}")
+    private String binder;
+
+    private final StreamBridge streamBridge;
+
+    public NotificationProducer(StreamBridge streamBridge) {
+        this.streamBridge = streamBridge;
+    }
+
```

Comment:
put a @Configuration here, as done in https://github.com/pagopa/idpay-payment/blob/main/src/main/java/it/gov/pagopa/payment/connector/event/trx/TransactionNotifierServiceImpl.java#L29 in order to pre-load the producer at startup